pkg/kafka_ingester/kafka_ingester.go

```diff
@@ -172,10 +208,53 @@ func (k *KafkaIngester) Run() {
 	}()
 }
 
+func getSchemaVersionFromHeaders(headers []kafka.Header) (string, bool) {
+	var (
+		found     bool
+		schemaVer string
+	)
+	for _, header := range headers {
+		if header.Key == schemaVersionMessageHeader {
+			schemaVer = string(header.Value)
+			found = true
```

Comment:
`return schemaVer, true`?

Reply:
Done as 89c41f8 within the primary PR https://github.com/seznam/slo-exporter/pull/48/files